packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -51,6 +53,13 @@ def jasmine_node_test(
         tags = tags,
     )
 
+    enableCoverage = False
```

Comment:
no need for the `enableCoverage` flag, we can just use coverage directly below.

Reply:
Ah, does `1` get converted to a boolean?
I've seen in some places `testonly = 1` and `testonly = True` being treated as equivilent

Reply:
it would appear not, I think it's reasonable to ask for just `True` or `False` as values here though